js/core/src/Blocks.svelte

```diff
@@ -315,7 +315,7 @@
 				) {
 					await app.post_data(
 						// @ts-ignore
-						`${app.config.root}/stream/${submit_map.get(dep_index).event_id()}`,
+						`${app.config.root + app.config?.api_prefix}/stream/${submit_map.get(dep_index).event_id()}`,
```

Comment:
I don't think we need the `?` since `app.config.root` would only work if the `app.config` object was defined
```suggestion
						`${app.config.root + app.config.api_prefix}/stream/${submit_map.get(dep_index).event_id()}`,
```